voorhoede/voorhoede-website
# Fix/landing page semantic titles

## What changes were made

- Add an sr-only h1 on the landing pages, that uses the title field from dato
- Add option to page header to exclude using semantic headings (to prevent duplicate h1 tags)
- Refactor page header to use setup syntax and remove unused slot

## How to test or check results

Check `h1` tag on /en/impact/digital-products-accessible-for-everyone. You can also check existing pages like the home page to see if the heading structure is untouched.

## Checks
